pages/story-inquiry.js

```diff
@@ -0,0 +1,63 @@
+import { renderRichText } from "../lib/richText";
+import {
+  getAuthorTestimonies,
+  getFrequentlyAnsweredQuestions,
+  getStoryInquiryPage,
+} from "../lib/api";
+import AuthorTestimony from "../components/AuthorTestimony";
+import QuestionAnswer from "../components/QuestionAnswer";
+
+export default function StoryInquiryPage({
+  storyInquiryPageData,
+  authorTestimonies,
+  questionsAndAnswers,
+}) {
+  const {
+    story_submission_title: storySubmissionTitle,
+    story_submission_description: storySubmissionDescription,
+    submission_guidelines_subtitle: submissionGuidelinesSubtitle,
+    submission_guidelines_description: submissionGuidelinesDescription,
+    faq_title: faqTitle,
+    author_testimony_title: authorTestimonyTitle,
+    donation_resources_subtitle: donationResourcesSubtitle,
+  } = storyInquiryPageData;
+
+  return (
+    <>
+      {renderRichText(storySubmissionTitle)}
+      {renderRichText(storySubmissionDescription)}
+      {renderRichText(submissionGuidelinesSubtitle)}
+      {renderRichText(submissionGuidelinesDescription)}
+      {renderRichText(donationResourcesSubtitle)}
+      {renderRichText(authorTestimonyTitle)}
+      {authorTestimonies.map((item, i) => {
+        return <AuthorTestimony key={i} authorTestimonyData={item} />;
```

Comment:
If we have an item id to use here instead of key it would be better, no worries if not